pypeit/core/telluric.py

```diff
@@ -755,7 +760,8 @@ def general_spec_reader(specfile, ret_flam=False):
         spect_dict = spec.spect_meta
         head = spec.head0
     else:
-        sobjs = specobjs.SpecObjs.from_fitsfile(specfile, chk_version=False)
+        sobjs = specobjs.SpecObjs.from_fitsfile(specfile, chk_version=chk_version)
+        # TODO: What bug?  Is it fixed now?  How can we test if it's fixed?
```

Comment:
I suspect we want to throw a reasonable user error here, and maybe make a note that this needs to be looked into.